stubs/pyOpenSSL/OpenSSL/crypto.pyi

```diff
@@ -143,8 +143,9 @@ class X509StoreContext:
     def verify_certificate(self) -> None: ...
 
 class X509StoreContextError(Exception):
+    errors: list[Any]
     certificate: X509
-    def __init__(self, message: str | bytes, certificate: X509) -> None: ...
+    def __init__(self, message: str | bytes, errors: list[Any], certificate: X509) -> None: ...
```

Comment:
OpenSSL's inline annotations have `message` as just `str`. Not sure where we got `bytes` from. That's a separate issue from this PR though.

Reply:
I changed it while I'm here.